deepchem/molnet/load_function/qm9_datasets.py

```diff
@@ -9,6 +9,7 @@
 
```

Comment:
t would be great to add unit tests and document the changes addressed by this QM9 update.